blueprint/core/blueprint_View.cpp

```diff
@@ -231,23 +249,10 @@ namespace blueprint
     {
         JUCE_ASSERT_MESSAGE_THREAD
 
-        if (props.contains(eventType) && props[eventType].isMethod())
+        if (auto *parent = findParentComponentOfClass<ReactApplicationRoot>())
         {
-            std::array<juce::var, 1> vargs { e };
-            juce::var::NativeFunctionArgs nfArgs (juce::var(), vargs.data(), static_cast<int>(vargs.size()));
-
-            try
-            {
-                std::invoke(props[eventType].getNativeFunction(), nfArgs);
-            }
-            catch (const EcmascriptEngine::Error& err)
-            {
-                if (auto* parent = findParentComponentOfClass<ReactApplicationRoot>())
-                    return parent->handleRuntimeError(err);
-
-                // If we coudln't find a parent that can handle it, rethrow
-                throw err;
-            }
+            std::vector<juce::var> vargs { getViewId(), eventType, e };
+            parent->engine.invoke("__BlueprintNative__.dispatchViewEvent", vargs);
```

Comment:
We should still catch the error here right?

Reply:
Ah yeah my mistake.

Reply:
Ah. Still have this left to do. 
Will sort that out.

Reply:
Just to clarify. We're talking about wrapping `engine.invoke` in `try/catch` here right?

Reply:
Should be sorted now

Reply:
Hm... maybe I'm missing something? I don't see that change

Reply:
Whoops, jk, I had gotten into a weird diff state on github. All good!